serverless-heaven/serverless-webpack
# Extends the --no-build option to serverless offline start

## What did you implement:

Closes #415

## How did you implement it:

Extended the check for --no-build to apply to the serverless offline cases.

## How can we verify it:

The tests have been modified to verify the functionality. I have also tested it with using serverless offline --no-build and verifying that the plugin is not invoked.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
